app/services/helpers/reduction-helper.js

```diff
@@ -0,0 +1,32 @@
+
+var dateFormatter = require('../date-formatter')
+
+module.exports.determineReductionsStatus = function (reductions) {
+  var active = []
+  var scheduled = []
+  var archived = []
+  var currentDate = new Date()
+  reductions.filter(function (reduction) {
+    if (currentDate.getTime() > reduction.reductionStartDate.getTime() &&
+            currentDate.getTime() < reduction.reductionEndDate.getTime()) {
+      active.push(reduction)
+    } else if (currentDate.getTime() < reduction.reductionStartDate.getTime()) {
+      scheduled.push(reduction)
+    } else if (currentDate.getTime() > reduction.reductionEndDate.getTime()) {
```

Comment:
Are we to have an archived flag in the reductions table.  ie active/scheduled deductions can be archived while the are still active/scheduled?

Reply:
We are to have status flags for Archived and Deleted records. If a record is in either of those two stages then it should not be in Active or Scheduled stage. I will update the query to reflect the status flag.